src/mayaMVG/qt/MVGCameraSetWrapper.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "MVGQt.hpp"
+#include "QObjectListModel.hpp"
+#include <maya/MColor.h>
+#include <maya/MFnSet.h>
+
+/**
```

Comment:
Move doc on the class (and not on the namespace) and maybe finish the sentence to avoid so much suspense ;)